### ISSUE TITLE
[Issue]: Cover art is being cut off

### ISSUE THREAD

**Issue by nlxdodge** [jellyfin/jellyfin#12190]
### Please describe your bug

See screenshots, this is the problem I am facing, but I am not sure if this is due to Jellyfin itself or any of the plugins retrieving the image.

What I have tried to fix:
- Click the three dots, click edit images, click delete Primary image delete, then refresh all metadata with replace existing images selected. It generated the same image dimension again, so I assume somewhere the cut is incorrect. Seeing this screenshot. 
- Re-create the docker container from scratch which I am running Jellyfin from.
- Re-creating the Anime collection and reloading the folder from the mounted path and scan the library again.
- Finally I set the image size to 2560 width max, set instead of the current top image fetcher the next in line to see if anything changed but still the same result after deleting and refreshing manually.

This is the browser inspector link:

`https://jellyfin.somewebsitethatithoughtof.com/Items/0c41907140d802bb58430fed7e2cd79e/Images/Primary?fillHeight=266&fillWidth=474&quality=96&tag=6d303ec0c946f722e1bdca7f1d089641`

Sizes look good just like the others, it's just that the bottom part is black.

### Reproduction Steps

1. Use version Jellyfin 10.9.2
2. Setup plugins and use default settings
3. Import some anime files and let everything finish
4. Folder image now looks like this

### Jellyfin Version

10.9.0

### if other:

10.9.2

### Environment

```markdown
- OS: Ubuntu 22.04.4 LTS
- Linux Kernel: GNU/Linux 5.15.0-112-generic x86_64
- Virtualization: Docker (via Compose file)
- Clients: Jellyfin Browse, Jellyfin Windows Client, Jellyfin Android App
- Browser: Firefox Nightly 128.0B8
- FFmpeg Version: `ffmpeg -version` not installed, and in the settings it's using `/usr/lib/jellyfin-ffmpeg/ffmpeg` 
- Playback Method: I assume Direct Play
- Hardware Acceleration: none
- GPU Model: none
- Plugins: AniDB, Anilist, AniSearch, AudioDB, Kitsu, MusicBrainz, OMDb, Open Subtitles, Studio Images, TMDb, TheTVDB.
- Reverse Proxy: Traefik
- Base URL: none
- Networking: network_mode: "host" is used in my docker compose file
- Storage: local with /mnt on linux
```

### Jellyfin logs

```shell
[logs.txt](https://github.com/user-attachments/files/16036110/logs.txt)
```

### Please attach any screenshots here

![image](https://github.com/jellyfin/jellyfin/assets/8994983/b6b0f566-79f3-42a0-8acb-27d9dc2bb45c)
![image](https://github.com/jellyfin/jellyfin/assets/8994983/63e538ba-ae95-4aaa-89e8-0db2c3b37f3b)
![image](https://github.com/jellyfin/jellyfin/assets/8994983/8ab86a22-df19-4620-8869-22619c355a3e)

Going to the browser and inspecting element also shows this as the raw image:

https://jellyfin.somewebsitethatithoughtof.com/Items/0c41907140d802bb58430fed7e2cd79e/Images/Primary?fillHeight=266&fillWidth=474&quality=96&tag=6d303ec0c946f722e1bdca7f1d089641

![image](https://github.com/jellyfin/jellyfin/assets/8994983/ac45a3ca-3bd8-4c75-b36e-e0033e4a8467)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Ashpan** [comment]
+1
I get the same issue with my Anime library only

**Comment by sjorge** [comment]
I'm guessing because the anilist plugin downloads banners as backdrops. So they are super wide and short.

I though there was an issue open for that om the plugin issue tracker, but I can't find it.

**Comment by nielsvanvelzen** [comment]
Those banners/backdrops work fine on the details page and other places they are used. I think we should change the library thumbnail generator instead to filter out the images with non-16:9 aspect ratios or scale & fit the image instead.

**Comment by nlxdodge** [comment]
My server disk went out of space so I had to reset my Jellyfin, but when I created the new Anime repository it now seems to be fixed. Was a fix already made in another commit?

**Comment by nlxdodge** [comment]
For me the issue is resolved with the latest Jellyfin version

**Comment by julian100004** [comment]
Has this really been fixed? If yes, what could I be doing wrong?

<img width="338" height="289" alt="Image" src="https://github.com/user-attachments/assets/40648686-fe50-4b69-b2cc-3c8e906c6359" />

**Comment by nlxdodge** [comment]
Indeed, I still see that the problem is there, but then again. The quickest fix is to set custom images.
And when using a custom client like Moonfin the problem is also not there.

**Comment by julian100004** [comment]
> Indeed, I still see that the problem is there, but then again. The quickest fix is to set custom images. And when using a custom client like Moonfin the problem is also not there.

Well, kind of 😅
But that's not really a fix, but just a workaround, is it?

**Comment by XylotoGhost** [comment]
I also face this issue, not only for an anime based library.

<img width="513" height="320" alt="Image" src="https://github.com/user-attachments/assets/cfec92b0-e5f3-4539-ba65-8bc2bcbd6156" />
